recipes/PyCm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pycm" %}
+{% set version = "7.5.1" %}
+{% set tag = "v7r5p1" %}
```

Comment:
Can you copy what I put together for Frv from [here](https://github.com/conda-forge/frv-feedstock/blob/0fde137a5637c8a1f2ee70c3d3f23c7a68ce2c21/recipe/meta.yaml#L2-L9)?

```suggestion
{% set version = "v7r5p1" %}

# handle Virgo version number format
{% set semver_version = version|
    replace('v', '', 1)|
    replace('r', '.', 1)|
    replace('p', '.', 1)
 %}
```